lms/resources/_js_config/__init__.py

```diff
@@ -195,6 +195,10 @@ def enable_lti_launch_mode(self, course, assignment: Assignment):
         # postMessage.
         self._config["hypothesisClient"] = self._hypothesis_client
 
+        self._config["hypothesisClient"][
+            "annotationMetadataJWE"
```

Comment:
Adding this here a top level config option. I don't have a preference on that direction or the naming, just an example.